streamalert_cli/terraform/generate.py

```diff
@@ -634,23 +608,6 @@ def generate_global_lambda_settings(
         tf_tmp_file (str): filename of terraform file, generated by CLI.
         message (str): Message will be logged by LOGGER.
     """
-    if conf_name == 'athena_partitioner_config':
-        # Raise ConfigError when user doesn't explicitly set `file_format`
-        # in `athena_partitioner_config` in conf/lambda.json when upgrade to v3.1.0.
-        file_format = get_data_file_format(config)
-
-        if not file_format or file_format not in ('parquet', 'json'):
```

Comment:
Can we keep this warning message here? There is chance that users may mis-config the file_format and we have warning message for the errors. But we need to update the message
```
            message = '[ERROR] Available values are "parquet" (preferred) and "json". You set the file_format to {} '.format(file_format)

```

Reply:
@chunyong-lin if you'd like this check, I'd rather have this be centralized as part of the config validation stuff here: https://github.com/airbnb/streamalert/blob/b7971a0020c10049256020e20e1850776ba4b77e/streamalert/shared/config.py#L338

Reply:
Fair!

Reply:
Let's do this later. I will open an issue about this.

Reply:
cool thanks!